front-end/src/components/optionSquare/optionSquare.module.scss

```
@@ -0,0 +1,44 @@
+@use '@styles/_fonts';
+@use '@styles/_variables';
+
+.optionSquare {
+    @include fonts.text-large;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    //use px for border
+    border-radius: 8px;
+    border: 1px solid var(--background);
+    background-color: var(--tertiary);
+    color: var(--text-color);
+    width: 13.435rem;
+    height: 11.25rem;
+
```

```suggestion
transition: filter 0.1s ease-in-out;
```
Would consider adding this to make the transition a bit less jarring